src/components/grid-aware/Modal/Modal.tsx

```diff
@@ -1,29 +1,38 @@
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 import ReactModal from "react-modal";
 
 import s from "./Modal.module.css";
 import closeIcon from "./close-icon.svg";
 
+type ModalProps = {
+  children: React.ReactNode;
+  isOpen: boolean;
+  setIsOpen: (x: boolean) => null;
+  ariaHideApp?: boolean; // we only ever set this to false for storybook
+  contentLabel: string;
+  noBezel?: boolean; // If true, then eliminate the padding and close button
+};
+
 const Modal = ({
   children,
   isOpen,
   setIsOpen,
   ariaHideApp,
```

Comment:
Regarding the issue with the default true prop, shouldn't you be able to assign an ES6 default argument value of `true`, like this?

```suggestion
  ariaHideApp = true,
```

This is basically what I did with the `isButton` prop in the Navigation component, which is optional: https://github.com/ShelterTechSF/sheltertech.org/blob/3cf755d7e289b5c9734e6ac3551eb3958d6f1e65/src/components/grid-aware/Navigation/BurgerMenu.tsx#L13

Reply:
Forgot about that! Very exciting feature 🎉